src/test/java/uk/ac/bath/se/DungeonControllerTest.java

```diff
@@ -5,6 +5,45 @@
 import static org.junit.Assert.assertTrue;
 
 public class DungeonControllerTest {
+    @Test
+    public void testCheckCollision() throws Exception {
+        DungeonController dungeonController = new DungeonController();
+        dungeonController.initialiseDungeonGame();
+        dungeonController.gridBounds[1][1] = Dungeon.BOUNDARY;
+        assertTrue(dungeonController.checkCollision(1, 1));
+
+        dungeonController.gridBounds[1][1] = Dungeon.EXIT;
+        assertTrue(dungeonController.checkCollision(1, 1));
+
+        dungeonController.gridBounds[1][1] = Dungeon.CHEST;
+        assertTrue(dungeonController.checkCollision(1, 1));
+
+        dungeonController.gridBounds[1][1] = 0;
+        assertTrue(!dungeonController.checkCollision(1, 1));
+    }
+
+    @Test
+    public void testBotCollisionShouldCollide() throws Exception {
+        DungeonController dungeonController = new DungeonController();
+        dungeonController.initialiseDungeonGame();
+        dungeonController.botPlayer.setxCoord(1);
+        dungeonController.botPlayer.setyCoord(1);
+        dungeonController.gridBounds[1][0] = Dungeon.BOUNDARY;
```

Comment:
Does this create a boundary on the left rather than above if gridbounds is drawn [y][x]?